GraphicsView/include/CGAL/Qt/Basic_viewer_qt.h

```diff
@@ -991,6 +991,13 @@ class Basic_viewer_qt : public CGAL::QGLViewer
         rendering_program_p_l.release();
     }
 
+    // Fix Z-fighting by drawing faces at a depth
+    GLfloat offset_factor;
+    GLfloat offset_units;
+    glGetFloatv(GL_POLYGON_OFFSET_FACTOR, &offset_factor);
+    glGetFloatv(GL_POLYGON_OFFSET_UNITS, &offset_units);
+    glPolygonOffset(0.1f, 0.9f);
```

Comment:
Be careful, this should only be called in case of 2D structures, as it might alter the 3D rendering of faces.

Reply:
@maxGimeno Thanks for pointing that out. I changed the code to offset only for 2D structures. Also, I did not change anything else in the code but the CI Test fails with message 'Differences in Polygonal_surface_reconstruction: Random_numbers are new and not committed'. Could you help me here?

Reply:
Yes, it was introduced by another  PR in master, and fixed by yet another. You should be able to easily fix that simply by  [merging master in your branch](https://github.com/CGAL/cgal/wiki/Git-FAQ#my-branch-is-really-old-i-would-like-to-update-it-to-get-latest-update-from-master).

Reply:
Thanks. I did the merge. All checks pass now.